Readme.md

```diff
@@ -52,15 +52,13 @@ ready for production use.
 
 ### Moya version vs Swift version.
 
-Because of the many Swift versions Moya supports, it might be confusing to
-find the version of Moya that you need. Below is a table that shows which version of Moya
-you should use for your Swift version.
+Below is a table that shows which version of Moya you should use for
+your Swift version.
 
 | Swift version | Moya version  |
 | ------------- | ------------- |
-| 3.X           | >= 8.0.0      |
-| 2.3           | 7.0.2 - 7.0.4 |
-| 2.2           | <= 7.0.1      |
+| 4.X           | >= 9.0        |
+| 3.X           | 8.0.0 - 8.0.5 |
```

Comment:
3.0 is probably not working from some of the last versions (like 8.0.3 and up maybe?). Not sure if all versions from 8.0 support Swift 3.1/3.2. It might be good idea to separate Swift 3.0, 3.1, 3.2 in this table.

Reply:
🤔, you might be right there. I wonder why the table wasn't updated when Swift 3.0 was dropped (if it was? Not sure)